### PR TITLE
Add alt text

### PR DESCRIPTION
**Pull request by dawei-wang**
Add alt text to scrollheight.png
Relates to mdn/content#21616

### CODE REVIEW

File: files/en-us/web/api/element/scrollheight/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The **`Element.scrollHeight`** read-only property is a
 measurement of the height of an element's content, including content not visible on the
 screen due to overflow.
 
-![](scrollheight.png)
+![Scroll height example featuring an ad for cat adoption](scrollheight.png)
```
Comment (estelle) [inline files/en-us/web/api/element/scrollheight/index.md:20]: This image conveys a lot of information that the sighted user can access but is not available to the screen reader user. It shows that the scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport. That should be conveyed in the alt text. The cat part isn't important.

**Review by estelle — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/element/scrollheight/index.md (hunk 1 above)
Comment (dawei-wang) [reply files/en-us/web/api/element/scrollheight/index.md]: @estelle I have added a more descriptive alt text.

File: files/en-us/web/api/element/scrollheight/index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The **`Element.scrollHeight`** read-only property is a
 measurement of the height of an element's content, including content not visible on the
 screen due to overflow.
 
-![](scrollheight.png)
+![We see an example element with four regions labeled padding-top, border-top, border-bottom, padding-bottom. The scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport.](scrollheight.png)
```
Comment (estelle) [inline files/en-us/web/api/element/scrollheight/index.md:20]:
```suggestion
![The user's veiwport is an element with four regions labeled padding-top, border-top, border-bottom, padding-bottom. The scroll height goes from the container's padding top to the end of the padding bottom, well beyond the top and bottom of the viewport.](scrollheight.png)
```

**Review by estelle — APPROVED** [review]
thank you

**Pull request merged**